src/BaseTemplates/ClassLibrary/project.json

```diff
@@ -1,21 +1,14 @@
 ﻿{
-  "version": "1.0.0-*",
-  "description": "$projectname$ Class Library",
-  "authors": [ "$username$" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
+    "version": "1.0.0-*",
+    "compilationOptions": {
```

Comment:
We can leave `compilationOptions` out entirely.  [It defaults to false](http://json.schemastore.org/project) unless specified otherwise.